client/src/components/Form/Form.spec.js

```diff
@@ -12,3 +15,11 @@ test('loads item', () => {
   const buttonItem = screen.getByText('Save')
   expect(buttonItem).toBeInTheDocument()
 })
+
+test('button has onClick functionality', () => {
+  render(<Form onAddQuestion={mockOnSubmit} />)
+
+  const buttonItem = screen.getByText('Save')
+  userEvent.click(buttonItem)
+  expect(mockOnSubmit).toHaveBeenCalled()
```

Comment:
Nice job! One test is missing though, and that is that we test that `mockOnSubmit` is called with what I type in the input element.